lua/dap.lua

```diff
@@ -347,6 +347,27 @@ function Session:_request_scopes(current_frame)
   end)
 end
 
+function Session:_goto(line, source, col)
```

Comment:
Doesn't matter much, but I think one of the following parameter ordering variants would make more sense.

```suggestion
function Session:_goto(line, col, source)
```

(source last because it's optional)



```suggestion
function Session:_goto(source, line, col)
```

(source -> line -> col, because line is inside the source, and col is inside the line)

Reply:
Actually, only the line important. This is the only information that is processed by real life adpaters. Everything else is optional-

Reply:
> . I tried to order by how probable I estimate that the given information is ever necessary. DAP protocol expects line and source. col is optional

Ok makes sense.  Let's keep it then. 

But I think if col is optional then the col + 1 can fail

Reply:
I put it right after the get_cursor call.